subsys/debug/stack_analyzer/Kconfig

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig STACK_ANALYZER
+	bool "Enable Stack analyzer functionality"
+	select INIT_STACKS
+	select THREAD_MONITOR
+	help
+	  Enable stack analyzer functionality and all the required modules.
```

Comment:
Maybe mention here that we just report about stack size usage, and not about what is in the stacks. E.g. the callstack.

E.g. 

"Enable Stack size analyzer"

Reply:
Done